OKR.md

```diff
@@ -3,7 +3,25 @@
 We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
 ## 2019 Q1
-TBD
+### Connectivity Magic
+* Gossipsub interops with go
+* Rendezvous Protocol is implemented
+* Connection manager supports dial backoff
+* QUIC transport in js
+* POC of WebBluetooth
+### Improved Performance
+* multistream 2.0 is implemented
+* DHT has stream support for searches
+* Improve DHT scalability
+* NAT manager core functionality is implemented (upnp, pmp, pcp, hole punching)
+### Improve Debuggability and Stability
+* go and js are interoperable
```

Comment:
I think we should define what we want to have tested. I don't think having "full" interop testing is going to be feasible in Q1, but I think we can at least tackle the core components and figure out our process for adding interop tests for new things, like the Rendezvous Protocol.

Reply:
Agree! DO you have a list suggestion?

Reply:
```suggestion
* go and js are interoperable
  * The dht has a full, passing suite of interop tests
  * Circuit Relay has a full, passing suite of interop tests
  * TCP and Websocket transports have full, passing suites of interop tests over SECIO and mplex
```

Here are my suggestions. We additionally have gossipsub, but it's covered by it's own OKR. I think floodsub is a nice to have, but with our current OKR list I think these are the higher priorities.